### DISCUSSION TITLE
brew install nvm (Error)

### DISCUSSION THREAD

**Discussion by Abu-laban** [Homebrew/discussions#1407] [Linux]
brew install nvm
==> Downloading https://ghcr.io/v2/linuxbrew/core/nvm/manifests/0.38.0
curl: (22) The requested URL returned error: 404

Error: Failed to download resource "nvm_bottle_manifest"
Download failed: https://ghcr.io/v2/linuxbrew/core/nvm/manifests/0.38.0

**Comment by wilkmaciej** [comment]
I have similar error
	brew install watch
	==> Downloading https://ghcr.io/v2/homebrew/core/ncurses/manifests/6.2
	Warning: Transient problem: timeout Will retry in 1 seconds. 3 retries left.
	Warning: Transient problem: timeout Will retry in 2 seconds. 2 retries left.  
	Warning: Transient problem: timeout Will retry in 4 seconds. 1 retries left.  
	##O=#  #                                                                      
	curl: (6) Could not resolve host: ghcr.io
	Error: Failed to download resource "ncurses_bottle_manifest"
	Download failed: https://ghcr.io/v2/homebrew/core/ncurses/manifests/6.2

ghcr.io domain appers to be down
    host ghcr.io
    Host ghcr.io not found: 2(SERVFAIL)

**Reply by carlocab** [reply]
This is likely a connection problem on your end. `ncurses` successfully installs for me:
```
root@9ad722ae65a4:/home/linuxbrew# brew install ncurses
==> Downloading https://ghcr.io/v2/linuxbrew/core/ncurses/manifests/6.2
######################################################################## 100.0%
==> Downloading https://ghcr.io/v2/linuxbrew/core/ncurses/blobs/sha256:fc5a0983a8afae8ef46945f3bec9dba9d66e81e879027d100d05088ea6f8
==> Downloading from https://pkg-containers-az.githubusercontent.com/ghcr1/blobs/sha256:fc5a0983a8afae8ef46945f3bec9dba9d66e81e8790
######################################################################## 100.0%
==> Pouring ncurses--6.2.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/ncurses/6.2: 3,901 files, 9.5MB
```

I get the same error for `nvm`, though.

**Reply by wilkmaciej** [reply]
Works now
I was checking website through browser, and it was returning DNS error. Now it just returns `{"errors":[{"code":"UNAUTHORIZED","message":"authentication required"}]}` so probably something was down for a while.

**Reply by Abu-laban** [reply]
Yes, Thanks for your support

**Reply by carlocab** [reply]
Accessing ghcr.io in a browser always requires authentication (any will do, though) so seeing that is expected. You can try accessing it again and it'll show you the same thing even if `brew install` is working for you.

**Comment by carlocab** [comment]
The `nvm` manifest seems to be missing for some reason. In the meantime, please try
```
brew install -s nvm
```
This will "build" from source, but that's just moving some files around, so it shouldn't take long.

**Reply by Abu-laban** [reply]
Thank you .
It's working fine I appreciate your support thanks.
